### DISCUSSION TITLE
old bookmarks not working after upgrade due to change is url param from `q` to `full`

### DISCUSSION THREAD

**Discussion by jkjha** [oracle/opengrok#4243] [Q&A]
I have recently upgraded my opengrok instance but the old url bookmarks not seems to be working... ealier URL format used to be "search?q=<query_param>" . 

```
https://opengrok.example.com/source/search?q=license&project=<project_name>&defs=
```

but now q is not part of URL instead it is replaced by full it seems...

https://opengrok.example.com/source/search?project=<project_name>&full=license&defs=&refs=&path=&hist=&type=&xrd=&nn=1

 Is there an easier way to fix it?

**Comment by divyankbhardwaj** [comment]
Hi I'm also trying to upgrade a opengrok can you please guide how you did it ?
